src/plugins/collapsibleColumns/collapsibleColumns.js

```diff
@@ -358,6 +366,22 @@ class CollapsibleColumns extends BasePlugin {
     this.hot.view.wt.wtOverlays.adjustElementsSize(true);
   }
 
+  /**
+   * Gets an array of physical indexes of hidden columns.
+   *
+   * @private
+   * @returns {number[]}
+   */
+  getHiddenColumns() {
+    return arrayReduce(this.#hiddenColumnsMap.getValues(), (indexesList, isHidden, physicalIndex) => {
```

Comment:
Can I propose changing the name to use `collapsed` instead of `hidden`? We're no longer using the `hiddenPlugin` and we can make it more collapsable than ever before.

```suggestion
  getCollapsedColumns() {
    return arrayReduce(this.#collapsedColumnsMap.getValues(), (indexesList, isHidden, physicalIndex) => {
```

Reply:
Fixed within https://github.com/handsontable/handsontable/pull/6828/commits/64c835d92a73728cb725815ec818e070874c3107. I would like to see the `getHiddenColumns` method implemented in the HiddenMap. The methods seem very common for that Map type and it's also used in the _HiddenColumns_ plugin. This will be one of many suggestions for improving index mapper.

Reply:
Good idea, same for `TrimMap`, we often ask for current config or return them in public API. Sounds like a nice addition @wszymanski 

Should be named `getHiddenIndexes`/`getTrimmedIndexes` because `HiddenMap` will be used for rows as well.